airflow/contrib/hooks/gcp_bigtable_hook.py

```diff
@@ -0,0 +1,232 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+from google.cloud.bigtable import Client
+from google.cloud.bigtable.cluster import Cluster
+from google.cloud.bigtable.instance import Instance
+from google.cloud.bigtable.table import Table
+from google.cloud.bigtable_admin_v2 import enums
+from airflow.contrib.hooks.gcp_api_base_hook import GoogleCloudBaseHook
+
+
+# noinspection PyAbstractClass
+class BigtableHook(GoogleCloudBaseHook):
```

Comment:
Can you add tests for this hook?

Reply:
@DariuszAniszewski  is on holidays, but let me add a comment on that - this is very similar story with Spanner hook, we are testing GCP hooks with System tests (and real GCP) rather than with mock unit tests. The logic in those hooks is really simple - usually simply calling the corresponding library method, so tests for each would be quite redundant - for example tests for update_cluster would test if Cluster constructor is called once and then if update() method is called once. That's still possible and easy to do (and we can still do it) but I don't think it adds a lot of value. 

Do you think it makes sense to have such tests @jmcarp ?